azure-pipelines.yml

```diff
@@ -176,9 +176,6 @@ jobs:
   - bash: ci/scripts/verible-lint.sh fpv
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: Verible FPV (Verilog lint)
-  - bash: ci/scripts/check-module-ids.sh
-    condition: eq(variables['Build.Reason'], 'PullRequest')
-    displayName: Check status_t Module IDs (Experimental)
     continueOnError: True
```

Comment:
I think you need to drop this line too? (To avoid doing unnecessary CI work for something that has failed Verible lint)

Reply:
Yes you are correct, thanks